migrating_from_ocp_3_to_4/premigration-checklists-3-4.adoc

```diff
@@ -83,6 +83,13 @@ NFS does not require a defined storage class.
 If an application uses an internal image in the `openshift` namespace that is not supported by {product-title} {product-version}, you can manually update the xref:../migrating_from_ocp_3_to_4/troubleshooting-3-4.adoc#migration-updating-deprecated-internal-images_troubleshooting-3-4[{product-title} 3 image stream tag] with `podman`.
 * [ ] The target cluster and the replication repository have sufficient storage space.
 * [ ] The xref:../authentication/understanding-identity-provider.adoc#supported-identity-providers[identity provider] is working.
+* [ ] You have checked the status of the `openshift.io/host.generated` annotation. 
```

Comment:
This is a checklist, so we want to keep the entries as brief as possible. "Check the status of x" is not really specific enough. I'll suggest text later.